test/spec/Feature/Query/UpdateSpec.hs

```diff
@@ -386,3 +386,71 @@ spec = do
           { matchStatus = 204
           , matchHeaders = [matchHeaderAbsent hContentType]
           }
+
+  context "limited update" $ do
+    it "works with the limit query param" $
+      request methodPatch "/projects?limit=2&select=id,name"
+          [("Prefer", "return=representation")]
+          [json| {"name": "PopOS"} |]
+        `shouldRespondWith`
+          [json|[
+            { "id": 1, "name": "PopOS" },
+            { "id": 2, "name": "PopOS" }]|]
+          { matchStatus = 200 }
+
+    it "works with the limit query param plus a filter" $
+      request methodPatch "/projects?limit=1&id=gt.3&select=id,name"
+          [("Prefer", "return=representation")]
+          [json| {"name": "PopOS"} |]
+        `shouldRespondWith`
+          [json|[{"id": 4, "name": "PopOS" }]|]
+          { matchStatus = 200 }
+
+    it "works with the limit and offset query params" $
+      request methodPatch "/projects?limit=2&offset=2&select=id,name"
```

Comment:
Offset working is on purpose, I don't see why not give users this flexbility that already comes for free.